plugin/mark.vim

```diff
@@ -1,5 +1,10 @@
+fun! HarpoonLeave()
```

Comment:
At some point, we could probably kill this file and the other .vim file.

Just use vim.cmd [[ ... ]]

Reply:
I agree.